### DISCUSSION TITLE
support tar.gz Google Photos archives instead of just zip

### DISCUSSION THREAD

**Discussion by andrewbanchich** [ente-io/ente#3041] [Enhancements]
title

**Comment by mnvr** [comment]
Just so that you're aware, we recommend unzipping these anyway if you wish to ensure the photo dates are correctly imported - https://help.ente.io/photos/faq/photo-dates#importing-from-google-takeout. So unless you have a single zip (or tar), having support for tgzs won't be as important as you might be thinking it to be.

**Reply by andrewbanchich** [reply]
Is there a reason Ente couldn't do this automatically for archives as well?

**Reply by mnvr** [reply]
It doesn't do it currently. I'm mentioning it lest you upload multiple and find out that some of you dates are off.

> To give you a sense of why it doesn't work naively - you could upload zip 1 today. You stop the app, and then you could upload zip 2 next week. There is a file in zip 1 whose metadata is there in zip 2. 
>
> Of course, these things can be handled in various ways, just that it doesn't work naively. We hope to improve the current approach at some point as we get time.

**Reply by andrewbanchich** [reply]
Oh yeah, just asking since if there's no technical reason not to then maybe it would be a nice feature request, since it's one less curveball for users / reason they need to read docs before using Ente.
